data/ui/curve_fitting.blp

```diff
@@ -0,0 +1,104 @@
+using Gtk 4.0;
+using Adw 1;
+
+template $GraphsCurveFittingWindow : Adw.Window {
+  modal: true;
+  title: _("Curve Fitting");
+  focus-widget: confirm_button;
+  default-width: 1000;
+  default-height: 650;
+  ShortcutController {
+    Shortcut {
+      trigger: "Escape";
+      action: "action(window.close)";
+    }
+  }
+
+  Box {
+    orientation: vertical;
+    Adw.HeaderBar {
+      show-end-title-buttons: false;
+      [start]
+      Button {
+        label: _("Cancel");
+        action-name: "window.close";
+      }
+
+      [end]
+      Button confirm_button {
+        label: _("Add fit to data");
+        styles ["suggested-action"]
+        clicked => $add_fit();
+      }
+    }
+
+    Box {
+      Box{
+        orientation: vertical;
+        vexpand: false;
+        width-request: 300;
+        Adw.EntryRow equation {
+          margin-top: 12;
+          margin-bottom: 12;
+          margin-start: 12;
+          margin-end: 12;
+          title: _("Equation");
+          text: _("a*x+b");
```

Comment:
```suggestion
```
This should be set by using the preference

Reply:
You are right, thanks. This was still left from a previous working build, forgot to remove this.